cluster/examples/kubernetes/ceph/sc.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: csi-rbd
+provisioner: rbd.csi.ceph.com
+parameters:
+  monitors: 172.30.211.217:6789,172.30.63.195:6789,172.30.213.13:6789  
```

Comment:
Might be nice to have a comment on this line that the mons have to be updated, but this will be automated soon when #2649 is fixed.